doc/nrf/ug_nrf5340.rst

```diff
@@ -56,10 +56,11 @@ Use this core for tasks that require high performance and for application-level
 The M33 TrustZone divides the application MCU into secure and non-secure domains.
 When the MCU boots, it always starts executing from the secure area.
 
-In Zephyr, the application core is divided into two different build targets:
+In Zephyr, the application core is built using one of the following build targets:
 
-* ``nrf5340dk_nrf5340_cpuapp`` for the secure domain
-* ``nrf5340dk_nrf5340_cpuapp_ns`` for the non-secure domain
+* ``nrf5340dk_nrf5340_cpuapp`` for the secure domain.
+* ``nrf5340dk_nrf5340_cpuapp_ns`` for the non-secure domain.
+  On selecting this build target, the device also builds an additional secure firmware component, :ref:`Secure Partition Manager (SPM) <secure_partition_manager>` or :ref:`Trusted Firmware-M (TF-M) <ug_tfm>`.
```

Comment:
s/the device also builds an additional secure firmware component/the build system also builds additional secure fomrware, ....

I would also add following:
The application core will first execute the secure firmware, that will then execute main firmware in non-secure domain.

Reply:
I assume this part is added only to the ``nrf5340dk_nrf5340_cpuapp_ns`` target bulletpoint

Reply:
Yes, this is added as part of nrf5340dk_nrf5340_cpuapp_ns build target.